### PR TITLE
generate prometheus config

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-network-runner/issues/528

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -885,7 +885,7 @@ func (s *server) PauseNode(ctx context.Context, req *rpcpb.PauseNodeRequest) (*r
 		return nil, err
 	}
 
-	if err := s.network.updateNodeInfo(); err != nil {
+	if err := s.network.UpdateNodeInfo(); err != nil {
```
Comment (dasconnor) [inline server/server.go:888]: Was this broken in main?

Comment (dasconnor) [reply server/server.go:888]: Are these two different functions?

**Review by dasconnor — APPROVED** [review]
Prometheus part looks good, just a little confused why you changed the `updateNodeInfo` function name

File: server/server.go (hunk 1 above)
Comment (felipemadero) [reply server/server.go:888]: Yup the upper one takes a look in network.go. By mistake the lower case one was used. The chance is to be uniform in the usage of the locked version by all server calls (AddNode, RestartNode, etc).

Comment (dasconnor) [reply server/server.go:888]: Can you open a ticket to change the name of one of those methods? This seems ripe for future bugs.

Comment (felipemadero) [reply server/server.go:888]: https://github.com/ava-labs/avalanche-network-runner/issues/548

**Pull request merged**